### PR TITLE
Create command/server information

### PR DESCRIPTION
**Pull request by Mikael-R**
# Create New Command

Create server-information command to show information about server!

### CODE REVIEW

File: src/commands/general/ServerInformation.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      embed.addField("ID", infos.id);
+    } else {
+      embed.setDescription("ERROR. This command is specific from servers!");
+    }
+
+    await message.reply(embed);
+  }
+}
+
+export default PingCommand;
```
Comment (danielsolartech) [inline src/commands/general/ServerInformation.ts:68]: Change this to `export default ServerInformationCommand;`

File: src/commands/general/ServerInformation.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+import { general } from "@Decorators/commandTypes";
+import command from "@Decorators/command";
+import Command from "@Commands/Command";
+import IMessage from "@Interfaces/IMessage";
+import { MessageEmbed } from "discord.js";
+
+@general
+@command({
+  name: "server-information",
```
Comment (danielsolartech) [inline src/commands/general/ServerInformation.ts:18]: Maybe can be only `server`

**Review by danielsolartech — CHANGES_REQUESTED** [review]

File: src/commands/general/ServerInformation.ts (hunk 1 above)
Comment (Mikael-R) [reply src/commands/general/ServerInformation.ts]: Fixed

File: src/commands/general/ServerInformation.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  description: "Show information's about server",
+})
+class PingCommand extends Command {
```
[the remark this answers is not in the document]
Comment (Mikael-R) [reply src/commands/general/ServerInformation.ts]: Fixed

**Review by danielsolartech — APPROVED** [review]

**Pull request merged**